### PR TITLE
fix tags sorting, closes OD-223 https://open-data-ed.atlassian.net/br…

### PR DESCRIPTION
**Pull request by sivang**
…owse/OD-223

### CODE REVIEW

**Comment by sivang** [comment]
@zelima have you merged ?

File: ckanext/ed/helpers.py [hunk 1]
```diff
@@ -275,7 +275,7 @@ def load_choices(field_meta=None):
 
 
 def alphabetize_dict(items, sort_by='display_name'):
-    sorted_dict = sorted(items, key=lambda x: x.get(sort_by))
+    sorted_dict = sorted(items, key=lambda x: x.get(sort_by)[0])
```
[the remark this answers is not in the document]
Comment (zelima) [reply ckanext/ed/helpers.py]: Hm... Got it - sorting on the first letter. But why is it needed?

[the remark this answers is not in the document]
Comment (zelima) [reply ckanext/ed/helpers.py]: Also What if multiple tags start on the same latter? How they are sorted? Are you sure this is a fix? I doubt...

So to fix this bug, I would go this way:

1. Add a failing test case (we have input on staging)
2. run tests and make sure it fails
3. Make changes in function
4. Run tests - passing?
   4.1 - Yes -> hooray
   4.2 -  Not -> Go back to 3

[the remark this answers is not in the document]
Comment (sivang) [reply ckanext/ed/helpers.py]: Why is it needed?
- Sorting in almost any programming languages happen in a non-dictionary way, this kind of enforces a dictionary-order.
- It really fixes it, but order of words with the same starting letter is not regarded.
- Next week.

[the remark this answers is not in the document]
Comment (zelima) [reply ckanext/ed/helpers.py]: @sivang I merged this as we are in a rush, but I really doubt this is the fix.  

> Sorting in almost any programming languages happen in a non-dictionary way, this kind of enforces a dictionary-order.

Not sure how this is related here. We are talking about whether to pass the whole string or the first letter as a key. And I don't know why they differ. This has nothing to do with dictionaries and programming languages 

>It really fixes it

I can't be sure this is the fix as there is no test checking it, neither can you. The existing test was passing even with the previous approach

**Pull request merged**